relayer-cli/src/commands/clear.rs

```diff
@@ -47,6 +78,14 @@ impl Runnable for ClearPacketsCmd {
             Err(e) => Output::error(format!("{}", e)).exit(),
         };
 
+        // Now that we have access to the counterparty chain at this point, we can
+        // match on `counterparty_key_name` and if a String exists, then override
+        // the counterparty chain's `key_name` in its own Config
+        if let Some(ref counterparty_key_name) = self.counterparty_key_name {
```

Comment:
I think you need to call `config` on an object of type `ChainHandle`, cannot call the method on the trait directly.

One way to do it is using the `chains` variable defined above (L71).

```rust
let dst_chain_cfg = chains.dst.config();
```